query/src/main/scala/filodb/query/exec/RemoteExec.scala

```diff
@@ -47,6 +48,29 @@ trait RemoteExec extends LeafExecPlan with StrictLogging {
                 querySession: QuerySession)
                (implicit sched: Scheduler): ExecResult = ???
 
+  private def applyTransformers(resp: QueryResponse,
+                                  querySession: QuerySession,
+                                  source: ChunkSource,
+                                  timeoutMs: Long)(implicit sched: Scheduler): QueryResponse = {
+    resp match {
+      case qr: QueryResult =>
+        val (obs, schema) =
+          applyTransformers(Observable.fromIterable(qr.result), qr.resultSchema, querySession, source)
+        val rvs = Await.result(obs.toListL.runToFuture, Duration(timeoutMs, TimeUnit.MILLISECONDS))
+        qr.copy(result = rvs, resultSchema = schema)
```

Comment:
quick question: is `copy` a data copy or a pointer copy ?

Reply:
Pointers only: https://docs.scala-lang.org/tour/case-classes.html#copying